src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -18,7 +17,12 @@ import scala.util.{Failure, Success}
 // Note that as per the language specification, this is instantiated lazily and only used when necessary (i.e. server mode)
 object CromwellServer extends DefaultWorkflowManagerSystem {
   val conf = ConfigFactory.parseFile(new File("/etc/cromwell.conf"))
-  private lazy val realDataAccess = DataAccess()
+
+  // NOTE: Currently the this.dataAccess is passed in to this.workflowManagerActor.
+  // The actor could technically restart with the same instance of the dataAccess,
+  // So, we're not shutting down dataAccess during this.workflowManagerActor.postStop() nor this.service.postStop().
+  // Not sure otherwise when this server is really shutting down, so this.dataAccess currently never explicitly closed.
+  // Shouldn't be an issue unless perhaps test code tries to launch multiple servers and leaves dangling connections.
```

Comment:
Yeah, we have some lifecycle stuff to figure out.  Thanks for documenting what's happening here.  :smile: